docs/CompatibleSoftware.rst

```diff
@@ -19,63 +19,191 @@ application, please contact ocio-dev as well.
 After Effects
 *************
 
-`Compositor - Adobe <http://www.adobe.com/products/aftereffects.html>`__
+Website: `<http://www.adobe.com/products/aftereffects.html>`__
 
-An OpenColorIO plugin is available for use in After Effects.
+Documentation :
 
-See `src/aftereffects
-<http://github.com/AcademySoftwareFoundation/OpenColorIO/tree/master/src/aftereffects>`__
-if you are interested in building your own OCIO plugins.
+- See `src/aftereffects <http://github.com/AcademySoftwareFoundation/OpenColorIO/tree/master/src/aftereffects>`__ if you are interested in building your own OCIO plugins.
 
-Pre-built binaries are also available for `download
-<http://www.fnordware.com/OpenColorIO>`__, courtesy of 
-`fnordware <http://www.fnordware.com>`__.
+- Pre-built binaries are also available for `download <http://www.fnordware.com/OpenColorIO>`__, courtesy of `fnordware <http://www.fnordware.com>`__.
+
+
+- Profile has been properly measured using a spectrophotometer), then choose your display. If the transform was approved on a different monitor, then maybe you
+
+
+Arnold (Autodesk)
+*****************
+
+Website : `<https://docs.arnoldrenderer.com/display/A5AFMUG/Color+Management>`__
```

Comment:
Thanks for updating the website link. I was also suggesting you move the link you had there previously to a new Documentation section under Arnold, instead of deleting it.

Reply:
Looks like it got put under Blender instead. It should be moved here and given the label "Color Management"